src/chainparams.cpp

```diff
@@ -978,6 +980,12 @@ class CLiquidV1Params : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 0;
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
 
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1554500; // November 1, 2021
```

Comment:
Isn't the current code (prior to this patch) buggy in that it leaves `consensus.vDeployments[TAPROOT]` uninitialised, so that it could be anything?

Reply:
Yep, oops. This has only been true on `master` and for a day or two (during which time we've been merging a lot of taproot-related PRs), but it shouldn't have happened nonetheless.

Reply:
BTW -- to clarify your comment on twitter, this won't cause Taproot to be enforced on the LIquid network (except in the sense that leaving these variables undefined is UB so theoretically anything could happen). It would, at worst, cause the functionaries to start signalling early. But even that won't happen, because ofc the functionaries are not running `master`.